File: LLM-Translations/GPT-4/codenet/C++/Java/s778828888.java

import java.util.Scanner;

public class s778828888 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int m = in.nextInt();
        int x = in.nextInt();
        int[] a = new int[105];

        for (int i = 0; i < m; ++i) {
            int temp = in.nextInt();
            a[temp] = 1;
        }

        int ansl = 0;
        int ansr = 0;
        int i;
        for (i = x; i <= n; ++i) {
            ansr += a[i];
        }
        for (i = x; i >= 0; --i) {
            ansl += a[i];
        }

        System.out.println(Math.min(ansl, ansr));
    }
}

// 

File: LLM-Translations/GPT-4/codenet/C++/Go/s778828888.go
package main

import (
	"fmt"
)

func main() {
	var n, m, x int
	var a [105]int
	fmt.Scanf("%d %d %d", &n, &m, &x)

	for i := 0; i < m; i++ {
		var temp int
		fmt.Scanf("%d", &temp)
		a[temp] = 1
	}

	ansl, ansr := 0, 0
	for i := x; i <= n; i++ {
		ansr += a[i]
	}
	for i := x; i >= 0; i-- {
		ansl += a[i]
	}

	fmt.Printf("%d\n", min(ansl, ansr))
}

func min(a, b int) int {
	if a < b {
		return a
	}
	return b
}

// 
